src/cryptsetup/cryptsetup.c

```diff
@@ -1122,10 +1122,10 @@ static int attach_tcrypt(
         return 0;
 }
 
-static char *make_bindname(const char *volume) {
+static char *make_bindname(const char *volume, const char *client) {
```

Comment:
maybe call the 2nd param `mechanism_id`?

Reply:
Refactored this so that only the suffix is passed and named it `token_type`.